packages/server/src/main/java/software/uncharted/terarium/hmiserver/proxies/mira/MIRAProxy.java

```diff
@@ -15,24 +15,24 @@
 
 @FeignClient(name = "mira-api", url = "${mira-api.url}")
```

Comment:
In the end its all the same, but other proxies have opted to add `path = "/api"` to the FeignClient declaration, but, in the end its the same

Reply:
how so?